examples/twr_aloha/twr_shell.c

```diff
@@ -218,3 +220,5 @@ int _twr_handler(int argc, char **argv)
     _print_usage();
     return -1;
 }
+
+SHELL_COMMAND(twr, "Two-way-ranging (TWR) cli", _twr_handler);
```

Comment:
`_twr_handler` can now be `static` (and actually must be `static` due to the leading underscore)